lib/task/import/csvAuthorityRecordImportTask.class.php

```diff
@@ -299,7 +306,14 @@ public function execute($arguments = [], $options = [])
                                 continue;
                             }
 
-                            if (null !== $relation = QubitActor::setTermRelationByName($subjects[$i], $options = ['taxonomyId' => QubitTaxonomy::SUBJECT_ID, 'culture' => $self->columnValue('culture')])) {
+                            $relation = QubitActor::setTermRelationByName(
```

Comment:
Most of this code is duplicated for places and subjects, and can be turned into a method called in both places.